Anomaly/Patches/ThingDefs_Races/Races_Entities_Misc.xml

```diff
@@ -192,9 +252,12 @@
 	<Operation Class="PatchOperationAdd">
 		<xpath>Defs/ThingDef[defName="Sightstealer"]/statBases</xpath>
 		<value>
-			<MeleeDodgeChance>1</MeleeDodgeChance>
-			<MeleeCritChance>1</MeleeCritChance>
-			<MeleeParryChance>1</MeleeParryChance>
+			<ArmorRating_Sharp>0.2</ArmorRating_Sharp>
+			<ArmorRating_Blunt>1</ArmorRating_Blunt>
+			<PainShockThreshold>90</PainShockThreshold>
```

Comment:
They are weak, sure,  but too weak in CE, 90% pain threshold is not going to make them infinitely tougher as even 100% pain threshold is only making a pawn take around 25% more damage to go down, 90% is not going to make things very hard on the player since downing pawns nonlethally in CE is a trivial task with melee and ranged targeting.